types/draft-js/index.d.ts

```diff
@@ -81,11 +82,11 @@ declare namespace Draft {
 
                 // Provide a map of inline style names corresponding to CSS style objects
                 // that will be rendered for matching ranges.
-                customStyleMap?: any;
+                customStyleMap?: Object;
```

Comment:
`Object` still allows `number` and `string`. Did you want `object`?

Reply:
I'm using this:
```ts
type StyleMap = {
  [styleName: string]: {
    [key: string]: string
  }
}
```
in it's place on my project. But I don't know if that structure is acceptable.

I think, @claudiopro can shed some light on that.

> Did you want `object`?

But yeah. `object` would be a more accurate representation. Should I push another commit?

Reply:
`DraftEditorProps` is unnecessarily vague:

https://github.com/facebook/draft-js/blob/master/src/component/base/DraftEditorProps.js#L172
```js
  customStyleMap?: Object,
```

We should prolly add a better type like the one you mention, which is already used by some callsites e.g.:

```js
+customStyleMap?: {[string]: {[string]: string}},
```

Reply:
`DraftEditorProps` is unnecessarily vague:

https://github.com/facebook/draft-js/blob/master/src/component/base/DraftEditorProps.js#L172
```js
  customStyleMap?: Object,
```

We should prolly add a better type like the one you mention, which is already used by some callsites e.g.:

```js
+customStyleMap?: {[string]: {[string]: string}},
```

Reply:
@claudiopro would you please check it now? This is much better now:

```ts
type DraftStyleMap = {
  [styleName: string]: React.CSSProperties
};
```

We can get CSSProperties autocomplete with it!

![Screenshot at 2019-06-17 23-18-54](https://user-images.githubusercontent.com/8050659/59623688-97b5cd80-9156-11e9-980e-1b21dc9912a6.png)

Reply:
This looks a great step in the right direction! Let me ensure that this is the recommended approach to type JS property names in TypeScript 🚀

Reply:
As far as I can tell the React team is not directly involved with TS type definitions, go ahead 👍

Reply:
I actually spotted @threepointone  in the commit history, he seems to be the right person to advise on types for CSS properties 😃

Reply:
`@types/react` uses `React.CSSProperties` for the `style` prop. And since the StyleMap from Draft.js will be consumed by `react`, I think it's alright...

Reply:
Sounds good to me, go ahead and merge!

Reply:
@DanielRosenwasser if it's alright, can you take a look and merge now?